scripts/build_load_data.py

```diff
@@ -0,0 +1,442 @@
+# coding: utf-8
+"""
+
+This rule downloads the load data from `Open Power System Data Time series <https://data.open-power-system-data.org/time_series/>` and interpolate the date to fill gaps. The resulting data saved as ``.csv`` file. The user can choose between two different data sources ("ENTSOE_transparency" or "ENTSOE_power_statistics"). 
+
+Relevant Settings
+-----------------
+
+.. code:: yaml
+
+    snapshots:
+
+    load:
+        url:
+        source:
+
+.. seealso::
+    Documentation of the configuration file ``config.yaml`` at
+    :ref:`load`
+
+Inputs
+------
+
+
+Outputs
+-------
+
+- ``resource/load.csv``:
+
+
+Description
+-----------
+
+The configuration options ``load: source:`` can be used to control which load data should be used in the model "ENTSOE_transparency" or "ENTSOE_power_statistics".
+
+
+"""
+
+import logging
+logger = logging.getLogger(__name__)
+from _helpers import progress_retrieve, configure_logging
+
+from pathlib import Path
+
+import pandas as pd
+
+
+def load_timeseries_opsd(years=None, fn=None, countries=None, source="ENTSOE_power_statistics"):
+    """
+    Read load data from OPSD time-series package version 2019-06-05.
+
+    Parameters
+    ----------
+    years : None or slice()
+        Years for which to read load data (defaults to
+        slice("2018","2019"))
+        
+    fn : file name
+    
+    countries : Countries for which to read load data.
+        
+    source : "ENTSOE_transparency" or "ENTSOE_power_statistics"
+
+    Returns
+    -------
+    load : pd.DataFrame
+        Load time-series with UTC timestamps x ISO-2 countries
+    """
+        
+    if countries is None:
+        countries = snakemake.config['countries']
+     
+    if source == 'ENTSOE_transparency':
+        load = (pd.read_csv(fn, index_col=0, parse_dates=True)
+                .loc[:, lambda df: df.columns.to_series().str.endswith('_load_actual_entsoe_transparency')]
```

Comment:
```suggestion
                .filter(like='_load_actual_entsoe_transparency')
```
or with regex ($ for end of line)
`.filter(regex='_load_actual_entsoe_transparency$')`